variants/fcos/okd-rpms/manifests/99-sca-disable.yaml

```diff
@@ -0,0 +1,8 @@
+kind: Secret
```

Comment:
ditto

Reply:
yes, it was just copy-pasted from registry/okd/okd-release:4.12.0;
while mitigations looks reasonable, then what is `sca` - i have no idea.
should remove them all?